docker-compose.yml

```
@@ -125,8 +125,10 @@ services:
     ports:
       - 9002:8080
     volumes:
-      - ./config/devel/rbac/permissions.json:/opt/app-root/src/rbac/management/role/permissions/compliance.json:z
-      - ./config/devel/rbac/definitions.json:/opt/app-root/src/rbac/management/role/definitions/compliance.json:z
+      - ./config/devel/rbac/definitions/compliance.json:/opt/app-root/src/rbac/management/role/definitions/compliance.json:z
+      - ./config/devel/rbac/permissions/compliance.json:/opt/app-root/src/rbac/management/role/permissions/compliance.json:z
+      - ./config/devel/rbac/definitions/inventory.json:/opt/app-root/src/rbac/management/role/definitions/inventory.json:z
+      - ./config/devel/rbac/permissions/inventory.json:/opt/app-root/src/rbac/management/role/permissions/inventory.json:z
```

Can't we map the whole folder instead?
```suggestion
      - ./config/devel/rbac:/opt/app-root/src/rbac/management/role:z
```